HW02_B_Mimic_starter.Rmd

```diff
@@ -91,7 +117,14 @@ The above graph lets you see some colobrlind friendly palettes. For the graph be
 Now using the above mpg dataset, make this graph
 
 ```{r graph 4 code}
-
+ggplot(mpg,aes(cty,class,color=class)) + 
+#Turn off the shape of outlying point
+geom_boxplot(color="black", outlier.shape=NA) + 
+geom_jitter(width=0,height=0.4) +
+scale_color_brewer(palette="Set2") +
+#I think the x-axis should be labeled as "City mpg" and y-axis should be labeled as "Car Class"
+labs(x="City mpg", y="Car Class", title="Horizontal Box Plot of City MPG and Car Class") + 
+theme(axis.line=element_line(color="black"), panel.grid.major=element_blank(), panel.grid.minor= element_blank(), panel.background=element_blank(), panel.border=element_blank()) 
 ```

```

Comment:
Good presentation! I also did not notice that the theme of the plot is different. However, is there a theme that can automatically match the sample graph's style?